angr/misc/bug_report.py

```diff
@@ -52,14 +46,14 @@ def print_versions():
         except ImportError:
             print("Python could not find " + m)
             continue
-        except Exception as e:
+        except Exception as e:  # pylint: disable=broad-except
             print("An error occurred importing %s: %s" % (m, e))
         print("Python found it in %s" % (python_filename))
         try:
             pip_package = python_packages.get(m, m)
             pip_version = pkg_resources.get_distribution(pip_package)
             print("Pip version %s" % pip_version)
-        except:
+        except Exception:  # pylint: disable-broad-except
```

Comment:
Could this be handled in a better way than disabling the linter? I imagine there are only 1-2 exceptions that might be raised here.

Reply:
Broad except is desired here, unless you can predict every possible error here; since this is generating data to send for a bug report if something goes wrong we should not the failure in the report rather than fail to make a report. I did remove the bare except though, since this `BaseExceptions` like `sys.exit` should propogate imo.